ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -482,24 +482,25 @@ EOF
 function enable_efa_pg_instance_config() {
```

Comment:
I mean remove `enable_efa_pg_instance_config` totally from `ipi-install-install`, but using `ipi-install-install-aws` instead, both `jq` and `yq` are available in that image.

Reply:
I got your meaning before, and I tried that, but met some issue, and I have no idea how to solve it.
https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/53218/rehearse-53218-periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-aws-ipi-efa-pg-f5/1802612273714302976 
https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/53218/rehearse-53218-periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-aws-ipi-efa-pg-f5/1802594765452087296
By the way, there is no `yq` in that image, got error "/bin/bash: line 250: yq: command not found" in https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/53218/rehearse-53218-periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-aws-ipi-efa-pg-f5/1802575797823737856
Another, I think updating in `ipi-install-install` should be ok, there is similar code in it. https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh#L455

Reply:
@yunjiang29 Does the updating in `ipi-install-install` look good to you? Or do you insist on updating in `ipi-install-install-aws`? If yes, can you please take a look at the error, it doesn't look something I updated.

Reply:
> I got your meaning before, and I tried that, but met some issue, and I have no idea why it fail.

@huali9 As the commit is gone, I can't see the changes, but as your changes is very simple, I don't think it's caused  by your changes, can you update again and re-trigger the job?

>it has yq-go instead.

yes, `yq-go` is the correct command, it's version 3.3 of yq.

> updating in ipi-install-install-aws

The reason is this step is applicable on AWS platform, ipi-install-install-aws is more proper for this.